defaults/main.yml

```diff
@@ -13,8 +13,14 @@ apache_admin_mail: root@localhost
 apache_server_name: myawesomeweb.com
 
 apache_sites_available_template_path: "{{ playbook_dir }}/templates/conf/sites-available"
+apache_files_path: "{{ playbook_dir }}/files"
```

Comment:
Maybe default value should point to a subfolder of files, something like files/apache/... This will avoid conflict if other role use file folder.

Reply:
Done ;)